ng-dev/misc/new-main-branch/cli.ts

```diff
@@ -33,19 +33,29 @@ async function handler() {
   const config = getConfig([assertValidGithubConfig]);
   const repoSlug = `${config.github.owner}/${config.github.name}`;
 
-  if (config.github.mainBranchName !== 'main') {
-    error(red('Current project is not part of the default branch renaming.'));
-    return;
-  }
-
   if (!hasLocalBranch(git, 'master')) {
     error(red('Local repository does not have a local branch named `master`. Aborting..'));
     return;
   }
 
   if (hasLocalBranch(git, 'main')) {
-    error(red('Local repository already has a branch named `main`. Aborting..'));
-    return;
+    console.warn(yellow('The new `main` branch is already fetched locally. In order to run'));
+    console.warn(yellow('this tool, the `main` branch needs to be non-existent locally.'));
+    console.warn('');
+    console.warn(yellow('The tool will re-fetch the `main` branch and configure it properly.'));
+
+    if (!(await promptConfirm('Do you want to proceed and delete the `main` branch?'))) {
```

Comment:
```suggestion
    if (!(await promptConfirm('Do you want to proceed and delete the local `main` branch?'))) {
```
might be good to explicitly mention that the _local_ branch will be deleted.